src/pipecat/transports/services/daily.py

```diff
@@ -843,6 +861,10 @@ def __init__(
         self._register_event_handler("on_participant_joined")
         self._register_event_handler("on_participant_left")
         self._register_event_handler("on_participant_updated")
+        self._register_event_handler("on_transcription_message")
```

Comment:
was this one missing?  it's not needed for recording, correct?  
(just curious, not blocking)

Reply:
Yes, it was missing, so I added it.

Reply:
This was on purpose, but since we added it we should also add

```
        await self._call_event_handler("on_transcription_message", message)
```

Reply:
Just created https://github.com/pipecat-ai/pipecat/pull/694